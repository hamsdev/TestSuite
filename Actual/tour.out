TRUE
CLIPS> (run)
Time?
day
Country?
India
Budget?
3
Travelling with?
family
Activity?
landmarks
(Zoo Sentosa Chinatown Mustafa) (0.5 0.5 0.5 0.5) (Rank 92 84.5 80 65)
places->Zoo Sentosa Chinatown Mustafa
CLIPS> (dribble-off)
