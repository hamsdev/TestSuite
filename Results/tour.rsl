tour.clp differences are as follows:
    13: places->Sentosa Zoo Chinatown Mustafa
    13: (Zoo Sentosa Chinatown Mustafa) (0.5 0.5 0.5 0.5) (Rank 92 84.5 80 65)
    14: CLIPS> (dribble-off)
    14: places->Zoo Sentosa Chinatown Mustafa
   Files do not have the same # of lines
