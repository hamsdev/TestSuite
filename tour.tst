(unwatch all)
(clear)
(setgen 1)
(load tour.clp)
(reset)
(open "testData.csv" testdata "r")
(dribble-on "Actual//tour.out")
(run)
(dribble-off)
(clear)
(open "Results//tour.rsl" tour "w")
(load "compline.clp")
(printout tour "tour.clp differences are as follows:" crlf)
(compare-files "Expected//tour.out" "Actual//tour.out" tour)
(close testdata)
(close tour)