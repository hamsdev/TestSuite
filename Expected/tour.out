TRUE
CLIPS> (run)
Time?
day
Country?
India
Budget?
3
Travelling with?
family
Activity?
landmarks
places->Sentosa Zoo Chinatown Mustafa
CLIPS> (dribble-off)